src/components/src/QuantitySelector/quantitySelector.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright © 2019 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import { getConfig } from '../utils/ConfigProvider';
+import './quantitySelector.less';
+
+let intl = { get: str => str };
+
+interface PurchaseDetailsMainProps {
+  /** Callback for when the + icon is clicked */
+  handleQuantityIncrement: any,
+  /** Callback for when the - icon is clicked */
+  handleQuantityDecrement: any,
+  /** Callback for when the quantity state changes  */
+  handleQuantityChange: any,
+  /** State decides whether to show a loading animation */
+  isLoading: any,
+  /** The quantity number to show */
+  itemQuantity: any,
+}
+
+function QuantitySelector(props: PurchaseDetailsMainProps) {
```

Comment:
Does it make sense to use this in cartlineitem eventually?

Reply:
yes where ever we have a quantity selector.  Seems like this might be copy and pasted code here.